inc/post-templates.php

```diff
@@ -63,17 +65,17 @@ function post_templates_dropdown() {
  * Filter the single template value, and replace it with
  * the template chosen by the user, if they chose one.
  */
-if( !function_exists( 'get_post_template' ) ) {
+if( ! function_exists( 'get_post_template' ) ) {
 	function get_post_template( $template ) {
 		global $post;
 		if ( is_object( $post ) ) {
 			$custom_field = get_post_meta( $post->ID, '_wp_post_template', true );
 		}
 		if ( !empty( $custom_field ) ) {
-			if ( file_exists( get_stylesheet_directory() . "/{$custom_field}") ) {
-				$template = get_stylesheet_directory() . "/{$custom_field}";
-			} else if ( file_exists( get_template_directory() . "/{$custom_field}") ) {
-				$template = get_template_directory() . "/{$custom_field}";
+			if ( file_exists( get_stylesheet_directory() . '/{$custom_field}' ) ) {
```

Comment:
@aschweigert Can we check that this works? I think that switching to single quotes here breaks the curly-braces interpretation: https://secure.php.net/manual/en/language.operators.string.php#110937